pkgs/tools/graphics/leela/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, pkgconfig, poppler }:
+
+stdenv.mkDerivation {
+  name = "leela-12.fe7a35a";
+
+  src = fetchFromGitHub {
+    owner = "TrilbyWhite";
+    repo = "Leela";
+    rev = "576a60185b191d3a3030fef10492fe32d2125563";
+    sha256 = "1k6n758r9dhjmc1pnpk6qzpg0q7pkq2hf18z3b0s2z198jpkg9s3";
+  };
+
+  buildInputs = [ pkgconfig poppler ];
+
+  installPhase = ''
+    make install PREFIX=$out MANDIR=$out/share/man
```

Comment:
You can use `installFlags` for setting PREFIX and MANDIR, instead of overwriting the whole installPhase.